### PR TITLE
Update Task-07 to work with current RIOT-OS API

### PR DESCRIPTION
**Pull request by Quintol**
Added msg_init_queue to Task 7.2.
Changed server creation to use macro.

### CODE REVIEW

File: task-07/README.md [hunk 1]
```diff
@@ -21,13 +21,22 @@ Note: on MAC use `bridge0` instead of `tapbr0`.
 
 ## Task 7.2: Extend `gnrc_minimal` application
 * Add the `gnrc_udp` module to the application's [Makefile](Makefile)
```
Comment (PeterKietzmann) [inline task-07/README.md]: While you're at it, could you adapt the link to the Makefile as well?

File: task-07/README.md [hunk 2]
```diff
@@ -21,13 +21,22 @@ Note: on MAC use `bridge0` instead of `tapbr0`.
...
+* To be able to receive packets, a packet buffer must be created using [msg_init_queue](https://doc.riot-os.org/group__core__msg.html#ga480e6f32c8ab18579b62a890f3fda2cd):
```
Comment (PeterKietzmann) [inline task-07/README.md]: That is not entirely correct. The `netreg ` (which is mentioned below) requires a queue in the default case (check description for [`gnrc_netreg_register()`](https://doc.riot-os.org/group__net__gnrc__netreg.html). This is also stated [here](https://doc.riot-os.org/group__net__gnrc.html) under "Architecture" point 1. The packet buffer has nothing to to with that. Please rephrase and reference one of the sources I linked (if you think it makes sense). Now that this is a requirement of `netreg`, wouldn't it make sense to explain it after the function is mentioned the first time (see some lines below)?

Comment (PeterKietzmann) [reply task-07/README.md]: Furthermore, please reduce the line width to something around 80-90.

**Review by PeterKietzmann — CHANGES_REQUESTED** [review]
@Quintol thanks a lot for your contribution!

**Comment by PeterKietzmann** [comment]
@Quintol ping

**Comment by Quintol** [comment]
@PeterKietzmann Forgot to commit my local changes... should be all there now

File: task-07/README.md [hunk 3]
```diff
@@ -20,14 +20,24 @@ ping6 <RIOT-IPv6-addr>%tapbr0
 Note: on MAC use `bridge0` instead of `tapbr0`.
 
 ## Task 7.2: Extend `gnrc_minimal` application
-* Add the `gnrc_udp` module to the application's [Makefile](Makefile)
+* Add the `gnrc_udp` module to the application's [Makefile](https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_minimal/Makefile)
```
Comment (PeterKietzmann) [inline task-07/README.md]: @Quintol thanks for addressing the other comments. Would you please split this line, for example before the Makefile reference? We should be ready to go afterwards

**Comment by PeterKietzmann** [comment]
Nice! Please squash to one commits

**Comment by Quintol** [comment]
If you can tell me how, i can to that

**Comment by PeterKietzmann** [comment]
Have a look at [this](https://github.com/RIOT-OS/RIOT/wiki/Git-cheatsheet) and [this](https://davidwalsh.name/squash-commits-git) guide or any other online tutorial for detailed instructions. Now that you did your changes on your local master branch the procedure would look somehow like:

- `gir rebase -i HEAD~3`
- prefix the last 2 commits with an "`s`" for "squash" instead of "pick"
- save, exit, and you will get to the commit description. Comment the last two messages with an "`#`"
- save and exit
- check if it worked with `git log`
- force push your changes with `git push -f`

**Comment by Quintol** [comment]
Done

**Review by PeterKietzmann — APPROVED** [review]
Thanks. ACK

**Pull request merged**